code/mob/living/critter/ai/flock/flocktasks.dm

```diff
@@ -586,22 +585,19 @@ butcher
 		var/mob/living/critter/flock/drone/F = holder.owner
 		usr = F // don't ask, please, don't
 		if(F?.set_hand(1)) // grip tool
-			if(!F.is_in_hands(container_target)) //if it's not in any of our hands, pick it up
```

Comment:
useful comment

Reply:
Idk, any way this could be more descriptive? Didn't really explain why item is dropped and then picked up

Reply:
We drop anything that isn't the target item, so as not to hit the target item with the thing we're holding. Comment could say that?